Documentation/docs-mobile/building-apps/build-items.md

```diff
@@ -129,6 +129,35 @@ multiple files, and they will be evaluated in no particular order (so don't
 specify the same environment variable or system property in multiple
 files).
 
+## AndroidGradleProjectReference
```

Comment:
I'm "wishy-washy" on this name.  It's kinda long, and feels like `@(AndroidGradeProject)` would work just as well.

Additionally, the `%(Include)` value is expected to be a *directory*, and thus doesn't mirror `@(ProjectReference)` and `@(Reference)` semantics, which have `%(Include)` refer to a *file*.

…but that said, `@(AndroidGradleProject)` referencing a directory doesn't feel right either, as a directory isn't a project?

Additionally, I'm not entirely sure this *should* reference a directory, as IIRC VS doesn't make this easy or straightforward (or possible?).  For example, open dotnet/java-interop within Visual Studio, then in the Solution Explorer navigate to [tools/java-source-utils](https://github.com/dotnet/java-interop/tree/51b784a235e0e443ff698e88a3464076912a1d09/tools/java-source-utils) and view the directory properties.  They're…not useful:

<img width="350" alt="image" src="https://github.com/user-attachments/assets/44d1641d-22b4-4c35-8fe7-01ae92d50d8d">

Meaning that as-is, there would be no way for customers to easily *use* this new Build action *unless* they hand-edit their `.csproj` files!  (Which is totally *me*, to be clear!  {What's an IDE?} That's *not* most customers.)

---

I think this property should be the (shorter) `@(AndroidGradleProject)`, *and* that `%(Include)` should refer to *a file*, `build.gradle`.  I believe that would allow reasonably straightforward setting of Build actions within the VS Solution Explorer.

Additionally, `@(AvailableItemName)` should be updated to include this new build action; see e.g.

https://github.com/dotnet/android/blob/958a330ab2378e5bf858dad707be720e4b3a5d90/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.AvailableItems.targets#L14-L30

Reply:
Renamed to `@(AndroidGradleProject)`, the Include attribute should now point to a build script rather than a directory, and @(AvailableItemName) has been updated.